config/boards/rock-5a.wip

```diff
@@ -0,0 +1,28 @@
+# Rockchip RK3588S octa core 4/8/16GB RAM SoC eMMC USB3 USB-C GbE
+declare -g BOARD_NAME="Rock 5A"
+declare -g BOARDFAMILY="rockchip-rk3588"
+declare -g BOOTCONFIG="rock-5a-rk3588s_defconfig"
+declare -g BOOT_SOC="rk3588"
+declare -g KERNEL_TARGET="legacy"
+declare -g FULL_DESKTOP="yes"
+declare -g BOOT_LOGO="desktop"
+declare -g BOOT_FDT_FILE="rockchip/rk3588s-rock-5a.dtb"
+declare -g BOOT_SCENARIO="spl-blobs"
+declare -g WIREGUARD="no"
+declare -g IMAGE_PARTITION_TABLE="gpt"
+declare -g SKIP_BOOTSPLASH="yes" # Skip boot splash patch, conflicts with CONFIG_VT=yes
+declare -g BOOTFS_TYPE="fat"
```

Comment:
I think we can remove this bootfs.

Reply:
Dropped `rock-5a` from this PR, we should do it separately.